Remove imgur environment variable

eleanorakh opened the pull request:
- Lita provides configuration which collects the imgur API key, so this PR removes the imgur API environment variable in favour of the lita config option.

yob commented:
The diff looks good!

My only suggestion is to include a change to the README that describes what configuration is required in `lita_config.rb`

eleanorakh committed:
Remove imgur environment variable
-Lita provides configuration which collects the imgur API key, so this PR removes the imgur API environment variable in favour of the lita config option.

eleanorakh committed:
Update README to include configuration instructions

yob commented:
:ship: it
